lib/tapjoy/ldap/group/delete.rb

```diff
@@ -20,8 +20,8 @@ def opts
           end
         end
 
-        def dn
-          @dn ||= "cn=#{opts[:name]},ou=Group,#{Tapjoy::LDAP::client.basedn}"
+        def distinguished_name
+          @distinguished_name ||= "cn=#{opts[:name]},ou=Group,#{Tapjoy::LDAP::client.basedn}"
         end
 
         def confirm
```

Comment:
not part of the PR but should the next line be 

```
puts "Confirm that you want to delete group #{opts[:name]} (yes/no)"
```

instead of 
```
puts "Confirm that you want to delete group #{opts[:group]} (yes/no)"
```